RELEASES.md

```diff
@@ -1,3 +1,141 @@
+Version 1.71.0 (2023-07-13)
+==========================
+
+<a id="1.71.0-Language"></a>
+
+Language
+--------
+
+- [Remove misleading target feature aliases, inappropriately connected to AVX-512.](https://github.com/rust-lang/rust/pull/107707/)
```

Comment:
```suggestion
```

avx512_target_feature is unstable.

Reply:
Ah, OK, I thought it would apply stability by the non-AVX names, but those are still unstable on the same feature gate.